docs/source/index.rst

```diff
@@ -9,7 +9,7 @@
 
 KBMOD (Kernel Based Moving Object Detection) is a GPU-accelerated framework for the detection of slowly moving asteroids within sequences of images. KBMOD enables users to detect moving objects that are too dim to be detected in a single image without requiring source detection in any individual image, nor at any fixed image cadence. KBMOD achieves this by “shift-and-stacking” images for a range of asteroid velocities and orbits without requiring the input images to be transformed to correct for the asteroid motion.
 
-.. Important:: If you use KBMOD for work presented in a publication or talk please help the project via proper citation or acknowledgement **what is proper citation or acknowledgement** In addition, if you use the known object matching functionality please cite either `IMCCE's SkyBoT VO tool <https://vo.imcce.fr/webservices/skybot/>`_ (Berthier et. al. 2006) or JPL’s SSD (Solar System Dynamics) `API service <https://ssd.jpl.nasa.gov/>`_ as appropriate.
+.. Important:: If you use KBMOD for work presented in a publication or talk please help the project via proper citation or acknowledgement `Whidden et. al. 2019 <https://ui.adsabs.harvard.edu/abs/2019AJ....157..119W/abstract>`_ and the Zenodo link for the code is `here <https://zenodo.org/record/7666852#.Y_ZtauzMKqU>`_. In addition, if you use the known object matching functionality please cite either `IMCCE's SkyBoT VO tool <https://vo.imcce.fr/webservices/skybot/>`_ (Berthier et. al. 2006) or JPL’s SSD (Solar System Dynamics) `API service <https://ssd.jpl.nasa.gov/>`_ as appropriate.
```

Comment:
Maybe it's my browser but the `Important` here doesn't render as bold or anything. It just looks like a one-word paragraph. Is there a better way to mark this in rst to bring attention to it?

Reply:
The entire paragraph is showing up in a callout box for me (when I build the HTML files and view them on my machine). I'm not sure what it isn't working for you. But since this was also the command used to render the previous versions, I think it is the one we want to use.